kin/transactions.py

```diff
@@ -120,9 +111,8 @@ def __init__(self, horizon_tx_response):
         """
         :param dict horizon_tx_response: the json response from an horizon query
         """
-        unpacker = Xdr.StellarXDRUnpacker(base64.b64decode(horizon_tx_response['envelope_xdr']))
-        envelop = unpacker.unpack_TransactionEnvelope()
-        self.tx = BaseTransaction.from_xdr_object(envelop.tx)
+        # Network_id is left as '' since we override the hash anyway
+        self.tx = decode_transaction(horizon_tx_response['envelope_xdr'], '', simple=False)
```

Comment:
if you're discussing an argument name in the comment, then explicitly state in the code using named arguments - i guess you're talking about the second argument to this function